bot/cogs/spam/spam.py

```diff
@@ -0,0 +1,87 @@
+import os
+import pathlib
+import time
+
+import discord
+from discord.ext import commands
+
+from ..variables import *
+
+spam_log_file = str(
+    pathlib.Path(
+        os.path.join(
+            os.path.abspath(os.getcwd()), "bot", "cogs", "spam", "spam_log.txt"
+        )
+    ).resolve()
+)
+
+
+class Spam(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.author.id == self.client.user.id:
+            return
+
+        ctx = await self.client.get_context(message)
+
+        _timestamp = str(int(time.time()))
+        _authorid = str(message.author.id)
+        _channelid = str(message.channel.id)
+
+        _log = _timestamp + "___" + _authorid + "___" + _channelid
```

Comment:
I would suggest to use `,` instead of `__` to make it CSV formatting.

No biggie though this way. If it works it works.